roles/pbx/tasks/main.yml

```diff
@@ -7,8 +7,8 @@
 
 - name: Install asterisk
   include_tasks: asterisk.yml
-  when: internet_available and is_ubuntu_18 and pbx_install and (not pbx_installed) and (not sugarizer_install) and (nodejs_version == "10.x")
+  when: internet_available and ((is_debian and ansible_distribution_major_version == "9") or is_ubuntu_18) and pbx_install and (not pbx_installed) and (nodejs_version == "10.x")
```

Comment:
@m-anish can you make this and Line 14 condition a bit more flexible, Debian 10 and other Ubuntus (like 19.04 coming in April) might actually work?

Reply:
I will once I have tested those platforms. But want to stick to this for now.

Reply:
> * consider `is_debian and (not is_rpi)`

Another option might be `is_debian` (just like `is_ubuntu` ideally) simplifying in the future anyway!?